src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -93,12 +93,34 @@ def get_age_distribution(self):
 
     def get_connection_types_prob_by_age(self):
         return {age: self.connection_type_prob_by_age_index[i] for i, age in enumerate(self.ages)}
-
+    
     # overriding hash and eq to allow caching while using un-hashable attributes
     __hash__ = object.__hash__
     __eq__ = object.__eq__
 
 
+    def get_required_adult_distributions(self):
+        students = 0
+        teachers = 0
+        for i in range(self.ages):
+            if self.ages[i] <= 18:
+                students += self.age_prob[i]*self.connection_type_prob_by_age_index[i][ConnectionTypes.School]
+            else:
```

Comment:
Needs to make sure that the ages are divided in such way.  At the version I a, looking at the ages are just 10, 40 or 70

Reply:
It is generalized to work with any ages.
The ages 10, 40, 70 are just the "default configuration", other values could have been loaded.